01_numeral_systems/practice/tasks.md

```diff
@@ -0,0 +1,32 @@
+# **Numeral system - tasks**
+
+1. Calculate:
+   ```
+    2543(10) = ?(2)
+    485(10)  = ?(2)
+    1024(10) = ?(2)
+
+    101011(2)    = ?(10)
+    110100101(2) = ?(10)
+    010111010(2) = ?(10)
+
+    101010110110(2) = ?(16)
+    2164(10)        = ?(16)
+
+    10101(2) | 110100(2) = ?(2)
+    1101(2)  & 1010(2)   = ?(2)
+    !101010101(2)        = ?(2)
+   ```
+---
+2. Check your result with the calculator on your computer (use programmer mode)
+---
+3. Take a look at the content of an arbitrary file with binary/hex editor.
+  - for Windows users: use Visual Studio -> File -> Open -> File -> Open with -> Binary Editor
+  - for Linux users: `$hexdump -C <file>`
+---
+4. Write a program in C++, that displays your name and faculty number.
```

Comment:
change that with which. I didnt knew the rule either. You can check it here:https://www.dailywritingtips.com/that-vs-which/